_config.yml

```diff
@@ -30,14 +30,18 @@ host: 127.0.0.1
 
 port: 4000
 # the port where the preview is rendered. You can leave this as is unless you have other Jekyll builds using this same port that might cause conflicts. in that case, use another port such as 4006.
+include:
+  - "*.md"
+  - "LICENSE"
 
 exclude:
+  - "*.yml"
+  - "**.html"
```

Comment:
Please check the warnings: "Build Warning: Layout 'post' requested in _posts/2018-02-02-welcome.md does not exist." https://travis-ci.org/kmindi/crypto-examples/builds/349551892#L1392 (even though the build succeded it gave no correct page, because the html files for the layout are not there and I guess this line is responsible). Correct this please.
With excluding using the hardcoded path it would still work I guess. But that's what we are trying to lose, right...